### PR TITLE
fixes #278 - Adds loklak wordcloud app

### LINKED ISSUE #278: Add loklak wordcloud app
Presently there is no app to create a word cloud from the data acquired from loklak. This app aims to generate a word cloud based on a query provided by the user.

### Features
User will provide a query word, query word can be a hashtag, a username for eg: @[username] or anything else. A word cloud will be generate using the tweets associated with the query word.
User may also use suggest API to generate the word cloud.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #278, adding loklak wordcloud app.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/LoklakWordCloud/
@kavithaenair @singhpratyush @Achint08 please review

This is the first basic version of loklak word cloud app. The basic feature of generating the word cloud has been implemented.
In the next iterations, filters (date, count, hashtags, mentions) will be added, error handling will be done and option to export the wordcloud to image and save will be added.

**Comment by Achint08** [comment]
Please see that the query data binding works fine.
screen shot 2017-07-30 at 3 10 35 am

**Comment by djmgit** [comment]
@singhpratyush thanks a lot for the suggestion.
I will clean the words, and solve the hashtag issue. I will address the the points related to word clicking feature and gradient in subsequent PRs as they are separate features and this is just the basic implementation of the app.
@vibhcool By different size of image, you are referring to the word cloud size right? That can be done :)
As of different colours, I will be implementing a better gradient.
Could please elaborate a little about the second point regarding distance :)

**Comment by vibhcool** [comment]
@djmgit , by distance, I meant words are separated by some parameter lets say it `d`, if this value is multiplied by a number taken from user if he wants, then different types of distribution of words can be observed. this will be fun :)

File: LoklakWordCloud/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+  <footer>
+  <script src="js/stopwords.js"></script>
+    <script src="js/script.js"></script>
```
Comment (kavithaenair) [inline LoklakWordCloud/index.html]: Indentation problem.

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/LoklakWordCloud/

@kavithaenair @singhpratyush @Achint08 please re-review, solved hashtag problem.

**Pull request merged**